magma/interface.py

```diff
@@ -1,7 +1,7 @@
 from itertools import chain
 from collections import OrderedDict
 from .conversions import array
-from .ref import AnonRef, InstRef, DefnRef, LazyDefnRef
+from .ref import AnonRef, InstRef, DefnRef, LazyDefnRef, ArrayRef, TupleRef, NamedRef
```

Comment:
looks like `AnonRef`, `ArrayRef`, and `TupleRef` imports are not used in this file